cirq-google/cirq_google/engine/engine_processor.py

```diff
@@ -126,29 +156,50 @@ def get_sampler(
             A `cirq.Sampler` instance (specifically a `engine_sampler.ProcessorSampler`
             that will send circuits to the Quantum Computing Service
             when sampled.
+        """
+        processor = self._inner_processor()
+        return processor_sampler.ProcessorSampler(
+            processor=self,
+            snapshot_id=snapshot_id,
+            device_config_name=(
+                device_config_name
+                if device_config_name
+                else processor.default_device_config_key.config_alias
+            ),
+            max_concurrent_jobs=max_concurrent_jobs,
+        )
 
-        Raises:
-            ValueError: If only one of `run_name` and `device_config_name` are specified.
-            ValueError: If both `run_name` and `snapshot_id` are specified.
+    def get_sampler(
```

Comment:
Following on from the above comment regarding the requirement of a device config to build a sampler, should we remove `get_sampler` from the processor and instead rely on explicitly grabbing the default configuration to generate the sampler?